src/commands/provar/automation/test/run.ts

```diff
@@ -45,6 +45,16 @@ export default class ProvarAutomationTestRun extends SfCommand<SfProvarCommandRe
       /* eslint-disable */
       const propertiesData = fileSystem.readFileSync(propertiesFilePath, { encoding: 'utf8' });
       const propertiesInstance = JSON.parse(propertiesData);
+      if (propertiesInstance?.testCase) {
+        propertiesInstance.testCase = propertiesInstance.testCase.map((testCasePath: string) =>
+          testCasePath.replace(/^\/tests\//, '/')
```

Comment:
what if `/tests/` is in between the path?

Reply:
regex catering the scenario